webrtc-audio-processing-sys/src/wrapper.hpp

```diff
@@ -18,6 +18,21 @@ const int NUM_SAMPLES_PER_FRAME = SAMPLE_RATE_HZ * FRAME_MS / 1000;
 
 struct AudioProcessing;
 
+struct OptionalDouble {
```

Comment:
Is there a way we can use the built-in Rust `Option` instead of exposing these types?

Reply:
Discussed offline and decided to map `Config` to its Rusty version in the parent crate. Leaving a TODO for now.